src/components/AuthForm.js

```diff
@@ -12,12 +52,16 @@ function AuthForm (props) {
       <div className="flex items-center justify-center px-4">
         <div className="flex flex-col w-[640px] gap-y-5">
           {
-            props.fields.map(field => {
-              return <InputForm fields={field} />
+            props.fields.map((field, index) => {
```

Comment:
'index' has been added as an additional argument to assign key values to each item/element in the map function.